lib/ansible/parsing/vault/__init__.py

```diff
@@ -143,48 +164,41 @@ def __init__(self, password):
         self.cipher_name = None
         self.b_version = b'1.1'
 
-    # really b_data, but for compat
-    def is_encrypted(self, data):
+    @staticmethod
+    def is_encrypted(data):
```

Comment:
<nod>  I already put deprecation messages in with the idea that they would go away in 2.4.  If we do code up a context-aware is_encrypted function, we'll then be able to bring them back with the new behaviour if we want.